src/Behat/Behat/Transformation/Context/Attribute/TrasnformationAttributeReader.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Transformation\Context\Attribute;
+
+use Behat\Behat\Context\Annotation\DocBlockHelper;
+use Behat\Behat\Context\Attribute\AttributeReader;
+use Behat\Behat\Transformation\Context\Annotation\TransformationAnnotationReader;
+use Behat\Transformation as Attribute;
+use ReflectionMethod;
+
+/**
+ * Reads transformation Attributes from the context class.
+ *
+ * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ */
+final class TrasnformationAttributeReader extends TransformationAnnotationReader implements AttributeReader
```

Comment:
```suggestion
final class TransformationAttributeReader extends TransformationAnnotationReader implements AttributeReader
```

Reply:
I had to read this twice to find the difference 🤣 Good spot!